nextflow/EVAImport/nextflow.config

```
@@ -10,6 +10,13 @@ profiles {
     process.memory = '2GB'
     process.cpus = 1
   }
+
+  slurm {
+    process.executor = 'slurm'
+    process.memory = '2GB'
+    process.cpus = 1
+    process.time = 48
```

Just to make this easier to understand:

```suggestion
    process.time = '48h'
```